src/index.js

```diff
@@ -0,0 +1,33 @@
+import './style.scss';
+import Weather from './weather';
+import displayInfo from './displayInfo';
+
+const searchForm = document.querySelector('form');
+const page = document.querySelector('body');
+const header = document.querySelector('header');
+const button = document.querySelector('button');
+const results = document.querySelector('.results-section');
+
+searchForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const cityName = document.querySelector('#city-name').value;
+
+  if (cityName === '') {
+    alert('please enter a valid city');
+  } else {
+    page.removeChild(header);
+    button.style.display = 'block';
+    results.style.display = 'block';
```

Comment:
- **_Keep all the styling logic in CSS files ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)_**